src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptCustomLanguageServerEditorImprovementsTest.java

```diff
@@ -166,4 +167,32 @@ export class Foo {
         assertEquals(enterBetweenBracesFileBody.replace(CARET, ""), document.getText());
         assertEquals(enterBetweenBracesFileBody.indexOf(CARET), caretModel.getOffset());
     }
+
+    // Verify LSPImprovedTextMateNestedBracesTypedHandler
+
+    public void testImprovedTextMateNestedBracesDisabledByDefault() {
```

Comment:
Confirms that the new TextMate-specific nested braces improvements are _disabled by default_ for non-user-defined language servers.